bodhi/server/renderers.py

```diff
@@ -98,11 +100,21 @@ def link_dict(obj):
                 return dict(href=request.route_url(route, **{param: obj[key]}))
             return link_dict
 
+        def describe_update(notes, builds):
```

Comment:
Can we add type annotations to describe what these arguments are? It seems like they might be callables based on how they are used below, but it would be good to describe their types with annotations and their purpose with a docblock. It would also be good to document the purpose of this function in a docblock.

Reply:
Yeah, I admit it is quite contorted indeed. I have taken example from the above `linker` function and I had had some problems myself to follow the code stream.

I've added a docstring now, I hope it's clear enough...

Reply:
@bowlofeggs Is there anything else that needs to be done in code to merge this PR?